Adapters set1 (#12)
SoftwareAG/sag-unattented-installations: 03.test/MSR/1011/msr-adapters/readme.md

# MSR 10.11 Deployer and ABE Test #1

> Purpose of this test is to verify the minimum installation of template MSR/1011/deployer, without the application of any fix.

Reviewer: ```suggestion
Purpose of this test is to verify the minimum installation of template MSR/1011/msr-adapters.
```
